File: modulo1/segmento3/apunte2/ejercicio1/ejercicio1.py
numero = int(input("Ingrese un numero entero para ver si es multiplo de 2: "))
resto = numero % 2

if resto != 0:
	print (str(numero) + " No es multiplo de 2 ")

else: 
	print (str(numero) + " es multiplo de 2 ")

File: modulo1/segmento3/apunte2 "Estructura de Decisión Doble"/ejercicio1/ejercicio1.java
import java.util.Scanner;
public class ejercicio1 
{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		System.out.print("Ingrese un numero: ");
		Integer numero = Integer.parseInt(teclado.nextLine());

		Integer resto = numero%2;

		if(resto !=0){
			System.out.println(numero.toString() + " no es multiplo de 2");
		}else{
			System.out.println(numero.toString() + " es multiplo de 2");

		}
	}
}

